build-support/bin/make/mac/10.13/4.2.1/build-make.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+source "$(git rev-parse --show-toplevel)/utils.v1.bash"
+
+set_strict_mode
+
+function fetch_extract_make_source_release {
+  local -r extracted_dirname="make-${MAKE_VERSION}"
+  local -r archive_filename="${extracted_dirname}.tar.gz"
+  local -r release_url="https://ftpmirror.gnu.org/gnu/make/${archive_filename}"
+
+  local -r downloaded_archive="$(curl_file_with_fail "$release_url" "$archive_filename")"
+  extract_for "$downloaded_archive" "$extracted_dirname"
+}
+
+function build_make_with_configure {
+  local -a configure_cmd_line=("$@")
```

Comment:
This seems like a good deal of ceremony over just a single line of `"$@"`.

Reply:
This was pilfered from the gcc script, which requires `--host` and `--target` arguments to `./configure`, in addition to being written as the first attempt of the new variety of binaries scripts (and therefore a bit clunky and overgeneralized). I will not look into fixing that for gcc but this indirection has no utility here, I think.